worlds/cvcotm/docs/en_Castlevania - Circle of the Moon.md

```diff
@@ -0,0 +1,116 @@
+
+Depending on what was chosen for the `completion_goal` option, your goal may be to defeat Dracula, complete the Battle Arena, or both.
+
+- `dracula`: Make it to the Ceremonial Room and kill Dracula's first and second forms to view the credits. The door to the
+Ceremonial Room can be set to require anywhere between 0-9 Last Keys to open it. If `require_all_bosses` is enabled, 8 keys
+will be required, and they will be guaranteed to be placed behind all 8 bosses (that are not Dracula).
+- `battle_arena`: Survive every room in the Battle Arena and pick up the Shinning Armor on the pedestal at the end. To make it
+easier, the `disable_battle_arena_mp_drain` option can be enabled to make the Battle Arena not drain your MP to 0, allowing
+DSS to be used. Reaching the Battle Arena in the first place requires finding the Heavy Ring and Roc Wing.
+- `battle_arena_and_dracula`: Complete both of the above-mentioned objectives. The server will remember which ones (if any) were
+already completed on previous sessions upon connecting.
+
+If you don't know what goal you have, there are two in-game ways you can check:
+
+- Pause the game, go to the Magic Item menu, and view the Dash Boots tutorial.
+- Approach the door to the first Battle Arena combat room and the textbox that normally explains how the place works will tell you.
+
+To see how many Last Keys are in the seed for you in total, there are also two in-game ways:
+
+- Pause the game, go to the Magic Item menu, and view the Last Key tutorial.
+- If you don't have any keys, touch the Ceremonial Room door before acquiring the necessary amount.
+
+
+## What items and locations get shuffled?
+
+Stat max ups, magic items, and DSS Cards are all randomized into the item pool, and the check locations are the pedestals
+that you would normally find the first two types of items on.
+
+The sole exception is the pedestal at the end of the Battle Arena. This location will always have Shinning Armor regardless
+of the chosen options.
+
+## Which items can be in another player's world?
+
+Stat max ups, magic items, and DSS Cards can all be placed into another player's world.
+
+The Dash Boots and Shinning Armor are not randomized in the item pool; the former you will always start with and the
+latter will always be found at the end of the Battle Arena in your own world. And depending on your goal, you may or may
+not be required to pick it up.
+
+## What does another world's item look like in Castlevania: Circle of the Moon?
+
+All items from other worlds will show up as the unused Map magic item. In-game, it does nothing when picked up besides setting
+the flag for the location so that the client can detect and send the location check.
+
+Upon sending an item, a textbox announcing the item being sent and the player who it's for will show up on-screen, accompanied
+by a sound depending on whether the item is filler, progression/useful, or trap-classified.
```

Comment:
In making progression/useful things play the same sound, I mainly wanted to be true to the game itself as everything in the game that is classified as progression and useful on AP's side normally plays the same sound when picked up (cards and magic items). The things considered "majors" by the randomizer, I guess you could say. The "minor" stat ups play their own sound when picked up, and for traps I decided to pick whatever since neither the regular game nor the randomizer has anything like this normally.